riposte-core/src/main/java/com/nike/riposte/server/handler/RequestInfoSetterHandler.java

```diff
@@ -21,10 +24,19 @@
  * This handler should come after {@link io.netty.handler.codec.http.HttpRequestDecoder} and {@link
  * SmartHttpContentCompressor} in the pipeline.
  *
+ * The request size is tracked and if it exceeds the configured global or a given endpoint's override, and exception
+ * will be thrown.
+ *
  * @author Nic Munroe
  */
 public class RequestInfoSetterHandler extends BaseInboundHandlerWithTracingAndMdcSupport {
 
+    private final int maxRequestSizeInBytes;
```

Comment:
Suggest renaming to `defaultMaxRequestSizeInBytes` or something else that indicates it's the default/global.